.rubocop_todo.yml

```diff
@@ -29,6 +29,7 @@ Metrics/ClassLength:
   Exclude:
     - 'app/models/claim/base_claim.rb'
     - 'app/controllers/external_users/claims_controller.rb'
+    - 'app/presenters/claim/base_claim_presenter.rb'
```

Comment:
non-blocking refactor, possibly using concern?

Reply:
not really enough to justify a concern in itself, especially as it is not shared (relying oon inheritance instead in the presenters generally). Is Tech debt generally from the size of the class. 

Reminds me that we can/should have tech debt tickets around fixing certain rubocop TODOs as well, including this one - will add to the tech debt document for now